client/src/tutorials/TictactoeSeahorse/pages/1.md

```diff
@@ -0,0 +1,25 @@
+# Tic-tac-toe Tutorial
+
+Today we'll be creating a classic 2 player game of Tic-tac-toe, also known as noughts and crosses [wikipedia](https://en.wikipedia.org/wiki/Tic-tac-toe). The game state will be stored on Solana including the grid, the current player's turn and eventually the winner. This will require two wallets, one for each player, these wallets will take turns interacting with the program.
+
+![Grid](/tutorials/tictactoe-seahorse/grid.png)
+
+Okay, so let's start with how to represent the game state. We do this in Seahorse with an account class which we'll call "Game". This will need to store the grid, the players, who's turn it is and if anyone has won. For the sake of simplification we'll be taking the classic tic-tac-toe 3x3 grid and flattening it into a traditional zero indexed array of length 9. The array will be filled with integers as we need to represent 3 states, "X", "O" or "empty" which we can represent with the integers "1", "2" and "0" respectively. 'u8' means an unsigned integer of 8 bits, i.e. a value from 0 to 255, inclusive.
```

Comment:
Can we use backticks instead of quotes when referencing code? For example, `Game` instead of "Game", and `u8` instead of 'u8'...